networkd/network_server.ml

```diff
@@ -844,6 +844,32 @@ module Bridge = struct
 				Sysfs.bridge_to_interfaces name
 		) ()
 
+	let get_physical_interfaces _ dbg ~name =
+		Debug.with_thread_associated dbg (fun () ->
+			match !backend_kind with
+			| Openvswitch ->
+				Ovs.get_real_bridge name
```

Comment:
Can you change the other function above to use the new `get_real_bridge` as well please.